lib/install.ts

```diff
@@ -14,3 +16,28 @@ function calculateTgetSelector(selector: string): string {
 }
 
 Cypress.Commands.add('tget', (selector, options) => cy.get(calculateTgetSelector(selector), options));
+
+Cypress.Commands.overwrite('within', function within(
+    originalWithinFn,
+    ...args
+) {
+    const aliases = cy.state('aliases');
+    const [withinSelector, navigationShorthand] =
+        ['____generatedNavigationWithin', '____generatedNavigation'].map(alias => aliases?.[alias]?.subjectChain[0]) as [string | undefined, object | undefined];
+
+    const fnIndex = args[1] instanceof Function ? 1 : 2;
+
+    // @ts-expect-error Typings are incorrect due to overrides
+    const originalCallback = args[fnIndex] as typeof args[1];
+
+    if ((withinSelector?.length ?? 0) > 0) {
+        // @ts-expect-error Typings are incorrect due to overrides
+        args[fnIndex] = <Subject>(subject: Subject) => originalCallback(navigationShorthand, subject);
+    }
+
+    return cy.wrap(originalWithinFn(...args), noLog)
+        .then(() => {
+            // eslint-disable-next-line no-underscore-dangle
+            delete aliases?.____generatedNavigationWithin;
```

Comment:
Suggestions for alternate names? When I add a transpile step and actually publish this, I could use the modern `#` for private without breaking older downstreams... but a Cypress alias like `@#genereatedNavigationWithin` to get rid of the underscores still feels odd.

Reply:
What specifically feels odd about that? And is `#` generally accepted as a modern standard enough to justify trying the new thing?

Reply:
Mostly that I had to disable a linting rule. 

Probably accepted enough: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields

Reply:
I'd say roll with the `#` standard, given that it looks like all browsers have it implemented now, and my own Googling indicated that enough people are using it to have a couple threads about having eslint work with the feature.

Reply:
How do you feel about `.as('cypressSelectorGenerator.#generatedNavigationWithin')` / `cy.get('aliases')['cypressSelectorGenerator.#generatedNavigationWithin']` That feels a little weird, too, but the chance of collision should be zero, and it makes it more explicit that we mean # as private and not any other meaning of the symbol in other contexts

Reply:
Roll with it!